.github/workflows/ci.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       - name: Jacoco test coverage
         run: |
           export TEST_AWS_COGNITO_IDTOKEN=${{ secrets.TEST_COGNITO_IDTOKEN }}
+          export AWS_SES_SOURCE=${{ secrets.AWS_SES_SOURCE }}
```

Comment:
In the future, put the changes impacting CI and CD file configuration into specific commits.